daemons/gptp/common/avbts_message.hpp

```diff
@@ -655,6 +688,72 @@ class FollowUpTLV {
 	int32_t getRateOffset() {
 		return cumulativeScaledRateOffset;
 	}
+
+    /**
+     * @brief  Updates the scaledLastGmFreqChanged private member
+     * @param  val Value to be set
+     * @return void
+     */
+    void setScaledLastGmFreqChange(int32_t val)
+    {
+        scaledLastGmFreqChange = PLAT_htonl(val);
+    }
+
+    /**
+     * @brief  Gets the current scaledLastGmFreqChanged value
+     * @return scaledLastGmFreqChange
+     */
+    int32_t getScaledLastGmFreqChange(void)
+    {
+        return scaledLastGmFreqChange;
+    }
+
+    /**
+     * @brief  Sets the gmTimeBaseIndicator private member
+     * @param  tbi Value to be set
+     * @return void
+     */
+    void setGMTimeBaseIndicator(uint16_t tbi)
+    {
+        gmTimeBaseIndicator = PLAT_htonl(tbi);
+    }
```

Comment:
I think this (Andrew's suggestion) is the convention that we decided on a while ago. We should be consistent.